doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -0,0 +1,512 @@
+:orphan:
+
+.. _ug_nrf54h20_ironside_se:
+
+IronSide Secure Element
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The IronSide Secure Element (IronSide SE) is a firmware for the Secure Domain of the nRF54H20 SoC that provides security features.
+It is provided as a precompiled binary, part of the nRF54H20 SoC bundle.
+
+IronSide SE provides the following features:
+
+* Global memory configuration (via UICR.MPCCONF)
```

Comment:
Note that here you're not using any formatting for peripherals and registers, yet later you do use ``. As per https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120291747/Typographical+conventions no formatting is required -- caps are enough.